<!DOCTYPE html>
<html class="no-js" lang="en">

<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <title>Loizos</title>
	<link rel="shortcut icon" href="images/me.ico" type="image/x-icon">
	<link rel="icon" href="images/me.ico" type="image/x-icon">
    <link href="https://fonts.googleapis.com/css?family=Lato:300,400,700,900" rel="stylesheet">
    <link rel="stylesheet" href="libs/font-awesome/css/font-awesome.min.css">
    <link href="css/bootstrap.min.css" rel="stylesheet">
    <link href="css/styles.css" rel="stylesheet">
</head>

<body>
    <div id="mobile-menu-open" class="shadow-large">
        <i class="fa fa-bars" aria-hidden="true"></i>
    </div>
    <!-- End #mobile-menu-toggle -->
    <header>
        <div id="mobile-menu-close">
            <span>Close</span> <i class="fa fa-times" aria-hidden="true"></i>
        </div>
        <ul id="menu" class="shadow">
            <li>
                <a href="#about">About</a>
            </li>
            <li>
                <a href="#experience">Experience</a>
            </li>
            <li>
                <a href="#education">Education</a>
            </li>
            <li>
                <a href="#publications">Publications</a> </li>
            <li>
            <li>
                <a href="#projects">Projects</a>
            </li>
            <li>
                <a href="#skills">Skills</a>
            </li>
            <li>
                <a href="#contact">Contact</a>
            </li>
        </ul>
    </header>
    <!-- End header -->

    <div id="lead">
        <div id="lead-content">
            <h1>Loizos Savva</h1>
            <h2>Bioinformatician</h2>
            <a href="https://drive.google.com/file/d/1RiTSz02vTehFXGr8sUdD8UvDpbLxRyjI/view?usp=sharing" class="btn-rounded-white">Download Resume</a>
        </div>
        <!-- End #lead-content -->

        <div id="lead-overlay"></div>

        <div id="lead-down">
            <span>
                <i class="fa fa-chevron-down" aria-hidden="true"></i>
            </span>
        </div>
        <!-- End #lead-down -->
    </div>
    <!-- End #lead -->

    <div id="about">
        <div class="container">
            <div class="row">
                <div class="col-md-4">
                    <h2 class="heading">About Me</h2>
                </div>
                <div class="col-md-8">
                    <p>
                        I am a Postdoctoral Researcher in Bioinformatics at the John Innes Centre with a strong background in computational chemistry. My PhD at Cardiff University focused on using molecular dynamics simulations to investigate proteins implicated in neurodegenerative diseases. Now, I apply my computational expertise to plant pathogen genomics, developing bioinformatics pipelines for analyzing next-generation sequencing data. My current work is centered on understanding and combating devastating crop diseases like wheat rust by providing real-time diagnostic solutions and supporting research into pathogen resistance.
                    </p>
                </div>
            </div>
        </div>
    </div>
    <!-- End #about -->

    <div id="experience" class="background-alt">
        <h2 class="heading">Experience</h2>
        <div id="experience-timeline">
            <div data-date="November 2023 – date">
                <h3>John Innes Centre</h3>
                <h4>Postdoctoral Researcher in Bioinformatics</h4>
                <p>
                    <ul>
                        <li> Finalised the development of the Snakemake-packaged MARPLE diagnostics pipeline for strain-level identification of wheat rust from field samples in remote locations, using data from MinION Nanopore sequencing devices.
                        <li> Developed the front-end user interface for interacting with the MARPLE pipeline, as well as adding metadata using a newly-developed barcoding system.
                        <li> Provided bioinformatics support in the Saunders’ Lab on research in wheat rust, working on NGS projects involving pathogen genomics and epigenetics.
                        <li> Collaborated with internal labs to identify susceptibility genes in wheat, to manage resistance to wheat blast.
                        <li> Completed a <i>de novo</i> genome assembly of <i>P. berberata</i> as part of a conservation-focused research project.
                        <li> Delivered bioinformatics workshops on Linux and R, online and in-person, to people with varying educational backgrounds.
                    </ul>
                </p>
            </div>

            <div data-date="January 2021 – August 2023">
                <h3>Cardiff University</h3>
                <h4>Graduate Demonstrator</h4>
                <p>
                    Demonstrated and tutored sessions in Computational, Physical and Inorganic Chemistry and Spectroscopy.
                </p>
            </div>

            <div data-date="June 2019 – July 2019">
                <h3>Emphasis Research Centre, University of Cyprus</h3>
                <h4>Research Intern</h4>
                <p>
                    <i>Project:</i> Detection of nicotine using HS-SPME-GC-MS and SERS.
                </p>
            </div>
        </div>
    </div>
    <!-- End #experience -->

    <div id="education">
        <h2 class="heading">Education</h2>
        <div class="education-block">
            <h3>Cardiff University</h3>
            <span class="education-date">Oct 2020 - Nov 2023</span>
            <h4>PhD in Computational Chemistry</h4>
            <p>
                <ul>
                    <li>
                        <b>Supervisor:</b> Prof. James A. Platts
                    </li>
                    <li>
                        <b>Thesis:</b> Computational examination of biomolecular systems related to Alzheimers and Parkinsons diseases
                    </li>
                    <li>
                        <b>Projects:</b> Assessed the effects of metal ions, mutations, pH and post-translational modifications on the folding and aggregation propensities of Amyloid-β and α-Synuclein, through the use of quantum and molecular mechanics, and semi-empirical methods.
                    </li>
                </ul>
            </p>
        </div>
        <!-- End .education-block -->

        <div class="education-block">
            <h3>University of Surrey</h3>
            <span class="education-date">Sept 2017 - Jun 2020</span>
            <h4>BSc(Hons) in Chemistry</h4>
            <p>
                <ul>
                    <li>
                        <b>Dissertation:</b> Adsorption of amino acids on platinum surfaces for enantioselective catalysis (using CASTEP simulations and Cyclic Voltammetry)
                    </li>
                    <li>
                        <b>Projects:</b> QSAR study on nicotinic agonists used in Alzheimers disease treatments; molecular evolutionary modelling of animal protein sequences.
                    </li>
                </ul>
            </p>
        </div>
        <!-- End .education-block -->
    </div>
    <!-- End #education -->

    <!-- Publications Section -->
   <div id="publications" class="background-alt">
        <h2 class="heading">Publications</h2>
        <div class="container">
            <div class="row">

				<div class="publication-block shadow-large">
                    <h3>Genetic analysis of the wheat yellow rust pathogen <em>Puccinia striiformis</em> f. sp. <em>tritici</em> in Bhutan reveals high levels of diversity
                    <span>Published: 2025</span>
                    <h4>Tshomo, Lhamo Y, Wouters R, Hodson DP, Saunders DGO, <b>Savva L</b>, Dema Y</h4>
                    <p>
                        <em>Zenodo</em>. DOI: <a href="https://doi.org/10.5281/zenodo.16743539" target="_blank">https://doi.org/10.5281/zenodo.16743539</a>
                    </p>
                    <a href="https://doi.org/10.5281/zenodo.16743539" target="_blank">Read Publication</a>
                </div>
				
				<div class="publication-block shadow-large">
                    <h3><em>k</em>-mer-based GWAS reveals a candidate avirulence gene and structural variation in <em>Puccinia triticina</em> linked to gain of <em>Lr20</em> virulence</h3>
                    <span>Published: 2025</span>
                    <h4>Tsushima A, Morier-Gxoyiya C, <b>Savva L</b>, Czajowski G, Wilderspin S, Czembor P, Holdgate S, Saunders DGO</h4>
                    <p>
                        <em>BMC Genomics</em>. DOI: <a href="https://doi.org/10.1186/s12864-025-12230-4" target="_blank">https://doi.org/10.1186/s12864-025-12230-4</a>
                    </p>
                    <a href="https://doi.org/10.1186/s12864-025-12230-4" target="_blank">Read Publication</a>
                </div>
				
                <div class="publication-block shadow-large">
                    <h3>First Large‐Scale Breakdown of <em>Yr15</em> Resistance to Wheat Yellow Rust (<em>Puccinia striiformis</em> f. sp. <em>tritici</em>) Recorded Globally</h3>
                    <span>Published: 2025</span>
                    <h4>Davis HR, Chia T, Rhodes H, Leaman C, Lewis CM, <b>Savva L</b>, Korolev A, Boyd L, Saunders DGO, Nellist CF, Kanyuka K</h4>
                    <p>
                        <em>New Disease Reports</em>. DOI: <a href="https://doi.org/10.1002/ndr2.70067" target="_blank">https://doi.org/10.1002/ndr2.70067</a>
                    </p>
                    <a href="https://doi.org/10.1002/ndr2.70067" target="_blank">Read Publication</a>
                </div>

                <div class="publication-block shadow-large">
                    <h3>A portable, nanopore-based genotyping platform for near real-time detection of <em>Puccinia graminis</em> f. sp. <em>tritici</em> lineages and fungicide sensitivity</h3>
                    <span>Published: 2025</span>
                    <h4><b>Savva L</b>, Bryan A, Vinopal D, Gonzalez-Navarro O, Kosgey Z, Ndungu Cyrus K, Horo Tola J, Danu Gutu K, Molla M, Alemayehu Y, Hodson DP, Saunders DGO</h4>
                    <p>
                        <em>BMC Genomics</em>. DOI: <a href="https://doi.org/10.1186/s12864-025-11428-w" target="_blank">https://doi.org/10.1186/s12864-025-11428-w</a>
                    </p>
                    <a href="https://doi.org/10.1186/s12864-025-11428-w" target="_blank">Read Publication</a>
                </div>

                <div class="publication-block shadow-large">
                    <h3>A recent shift in the <em>Puccinia striiformis</em> f. sp. <em>tritici</em> population in Serbia coincides with changes in yield losses of commercial winter wheat varieties</h3>
                    <span>Published: 2024</span>
                    <h4>Župunski V, <b>Savva L</b>, Saunders DGO, Jevtić R</h4>
                    <p>
                        <em>Frontiers in Plant Science</em>. DOI: <a href="https://doi.org/10.3389/fpls.2024.1464454" target="_blank">https://doi.org/10.3389/fpls.2024.1464454</a>
                    </p>
                    <a href="https://doi.org/10.3389/fpls.2024.1464454" target="_blank">Read Publication</a>
                </div>

                <div class="publication-block shadow-large">
                    <h3>Computational investigation of copper-mediated conformational changes in α-synuclein dimer</h3>
                    <span>Published: 2024</span>
                    <h4><b>Savva L</b>, Platts JA</h4>
                    <p>
                        <em>Physical Chemistry Chemical Physics</em>. DOI: <a href="https://doi.org/10.1039/D3CP04697D" target="_blank">https://doi.org/10.1039/D3CP04697D</a>
                    </p>
                    <a href="https://doi.org/10.1039/D3CP04697D" target="_blank">Read Publication</a>
                </div>

                <div class="publication-block shadow-large">
                    <h3>Exploring the impact of mutation and post-translational modification on α-Synuclein: Insights from molecular dynamics simulations with and without copper</h3>
                    <span>Published: 2023</span>
                    <h4><b>Savva L</b>, Platts JA</h4>
                    <p>
                        <em>Journal of Inorganic Biochemistry</em>. DOI: <a href="https://doi.org/10.1016/j.jinorgbio.2023.112395" target="_blank">https://doi.org/10.1016/j.jinorgbio.2023.112395</a>
                    </p>
                    <a href="https://doi.org/10.1016/j.jinorgbio.2023.112395" target="_blank">Read Publication</a>
                </div>

                <div class="publication-block shadow-large">
                    <h3>Evaluation of implicit solvent models in molecular dynamics simulation of α-Synuclein</h3>
                    <span>Published: 2023</span>
                    <h4><b>Savva L</b>, Platts JA</h4>
                    <p>
                        <em>Journal of Biomolecular Structure and Dynamics</em>. DOI: <a href="https://doi.org/10.1080/07391102.2022.2082534" target="_blank">https://doi.org/10.1080/07391102.2022.2082534</a>
                    </p>
                    <a href="https://doi.org/10.1080/07391102.2022.2082534" target="_blank">Read Publication</a>
                </div>

                <div class="publication-block shadow-large">
                    <h3>How Cu(II) binding affects structure and dynamics of α-synuclein revealed by molecular dynamics simulations</h3>
                    <span>Published: 2023</span>
                    <h4><b>Savva L</b>, Platts JA</h4>
                    <p>
                        <em>Journal of Inorganic Biochemistry</em>. DOI: <a href="https://doi.org/10.1016/j.jinorgbio.2022.112068" target="_blank">https://doi.org/10.1016/j.jinorgbio.2022.112068</a>
                    </p>
                    <a href="https://doi.org/10.1016/j.jinorgbio.2022.112068" target="_blank">Read Publication</a>
                </div>

                <div class="publication-block shadow-large">
                    <h3>Forcefield evaluation and accelerated molecular dynamics simulation of Zn(II) binding to N-terminus of amyloid-β</h3>
                    <span>Published: 2021</span>
                    <h4>Alshammari N, <b>Savva L</b>, Kennedy-Britten O, Platts JA</h4>
                    <p>
                        <em>Computational Biology and Chemistry</em>. DOI: <a href="https://doi.org/10.1016/j.compbiolchem.2021.107540" target="_blank">https://doi.org/10.1016/j.compbiolchem.2021.107540</a>
                    </p>
                    <a href="https://doi.org/10.1016/j.compbiolchem.2021.107540" target="_blank">Read Publication</a>
                </div>
            </div>
        </div>
    </div>
    <!-- End #publications -->

    <div id="projects" class="background-alt">
        <h2 class="heading">Projects</h2>
        <div class="container">
            <div class="row">
                <div class="project shadow-large">
                    <div class="project-image">
                        <img src="images/marple.png" />
                    </div>
                    <!-- End .project-image -->
                    <div class="project-info">
                        <h3>MARPLE</h3>
                        <p>
                            MARPLE (Mobile And Real-time PLant disEase) is a portable diagnostics pipeline for the rapid, in-field identification of wheat rust pathogens. Packaged using Snakemake, it utilizes data from Nanopore MinION sequencing devices to provide strain-level identification and assess fungicide sensitivity directly from infected samples, facilitating timely disease management in remote agricultural settings.
                        </p>
                        <a href="https://marple-diagnostics.org/">View Project</a>
                    </div>
                    <!-- End .project-info -->
                </div>
                <!-- End .project -->
            </div>
        </div>
    </div>
    <!-- End #projects -->

    <div id="skills">
        <h2 class="heading">Skills</h2>
        <ul>
            <li>Python</li>
            <li>R</li>
            <li>Nextflow</li>
            <li>Snakemake</li>
            <li>Bash/Shell</li>
            <li>Unix/Linux</li>
        </ul>
    </div>
    <!-- End #skills -->

    <div id="contact">
        <h2>Get in Touch</h2>
        <div id="contact-form">
            <form method="POST" action="https://formspree.io/f/mnnvepdk">
                <input type="hidden" name="_subject" value="Contact request from personal website" />
                <input type="email" name="_replyto" placeholder="Your email" required>
                <textarea name="message" placeholder="Your message" required></textarea>
                <button type="submit">Send</button>
            </form>
        </div>
        <!-- End #contact-form -->
    </div>
    <!-- End #contact -->

    <footer>
        <div class="container">
            <div class="row">
                <div class="col-sm-5 copyright">
                    <p>
                        Copyright &copy; <span id="current-year">2026</span> Loizos Savva
                    </p>
                </div>
                <div class="col-sm-2 top">
                    <span id="to-top">
                        <i class="fa fa-chevron-up" aria-hidden="true"></i>
                    </span>
                </div>
                <div class="col-sm-5 social">
                    <ul>
                        <li>
                            <a href="https://github.com/502107" target="_blank"><i class="fa fa-github" aria-hidden="true"></i></a>
                        </li>
                        <li>
                            <a href="https://linkedin.com/in/LoizosSavva" target="_blank"><i class="fa fa-linkedin" aria-hidden="true"></i></a>
                        </li>
                    </ul>
                </div>
            </div>
        </div>
    </footer>
    <!-- End footer -->

    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
    <script src="js/scripts.min.js"></script>
</body>

</html>
